spec/features/user_behaviours_spec.rb

```diff
@@ -53,7 +53,7 @@
       sign_out
       visit city_path(@user.home_city)
       click_link('5 spots left')
-      fill_in :name, with:  @user.name
+      fill_in 'tea_time_nickname', with:  @user.name
```

Comment:
Capybara#fill_in uses the words in label of given field, id, or name of field and is case sensitive.

http://www.rubydoc.info/github/jnicklas/capybara/Capybara/Node/Actions:fill_in